### ISSUE TITLE
Showing the Ice Icon and the hidden indicators together isn't nessecary

### ISSUE THREAD

**Issue by riegaz** [jordanbaird/Ice#25]
It seems that when items are shown, each category has its indicator arrow shown beside the Ice icon itself. This isn't needed, as simply showing the hidden icons is enough. This saves some space in the crowded menubar.

![CleanShot 2024-03-05 at 09 53 31@2x](https://github.com/jordanbaird/Ice/assets/1749401/d03fe57c-5c0a-43f1-8fa5-0d76298e7d79)

**Comment by jordanbaird** [comment]
The indicator arrows are actually required due to the way Ice hides menu bar items. I was just experimenting with a possible way around this, but it looks like it will require a significant amount of reimplementation. That's not to say it won't happen, it will just take some time.

**Comment by riegaz** [comment]
I see. As I don't know how it is implemented I just can tell that it makes sense to have separate areas for hidden and always hidden items. If the indicators just won't show, e.g., 1px x 1px images, that would already be enough. They don't have any function if items are shown on clicking the menubar, right?

**Comment by jordanbaird** [comment]
That was actually the first thing I tried. Unfortunately, it looks like macOS enforces a minimum size limit on menu bar items, so even with a size set to 0, there's still a decent amount of extra space. I think the solution is going to be to remove the items completely when the sections are visible, and reinsert them when hidden. I tried it, and it works well, but breaks several pieces of existing functionality, so that's where the reimplementation comes in.
